tests/e2e/auth.setup.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import { test as setup, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { customer, admin } from './test-data/data/data';
+
+const adminFile = 'playwright/.auth/admin.json';
+
+setup( 'authenticate as admin', async ( { page } ) => {
+	// Authentication steps.
+	await page.goto( '/my-account' );
+	const isLoggedOut = await page
+		.getByLabel( 'Username or email address' )
+		.isVisible();
+
+	if ( isLoggedOut ) {
+		await page
+			.getByLabel( 'Username or email address' )
+			.fill( admin.username );
+		await page.getByLabel( 'Password' ).fill( admin.password );
+		await page.getByRole( 'button', { name: 'Log in' } ).click();
+		// Wait until the page receives the cookies.
+		//
+		// Sometimes login flow sets cookies in the process of several redirects.
+		// Wait for the final URL to ensure that the cookies are actually set.
```

Comment:
```suggestion
		// Sometimes login flow sets cookies in the process of several redirects.
		// Wait for the final URL to ensure that the cookies are actually set.
```

Let's tidy up the comments a little bit.